spec/features/users_cannot_navigate_to_areas.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+describe 'as a visitor to the the app i should get a 404 error' do
+  it 'navigates to any /profile, /admin or /dashboard path' do
+    paths = profile_path,
```

Comment:
I don't think this will work how you want it to, i think it still needs `[ ]` around it to work correctly

Reply:
i brought this up with mary over a call. she seems to think it will work but i am skeptical. lol we were unsure if capybara would understand it.

Reply:
i do think we should try wrapping it in an array bracket.

Reply:
i tried it in just a basic ruby file and it gave me an error saying invalid number of arguements